scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,259 @@
+  fonts-oradano-mincho-gsrr \
+  fonts-kiloji \
+  fonts-mikachan \
+  fonts-seto \
+  fonts-yozvox-yozfont \
+  fonts-aoyagi-kouzan-t \
+  fonts-aoyagi-soseki \
+  fonts-kouzan-mouhitsu
+#   ttf-mscorefonts-installer
+
+# Install Python 3.7
+RUN apt-get update && apt-get install -y \
```

Hadolint

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)